File: java/006/Euler.java
public class Euler {
    public static void main(String[] args) {
        long sumOfSquares = 0;

        long sum = 0;

        long squareOfSum = 0;

        for (long i = 1; i <= 100; i++) {
            sumOfSquares += i * i;

            sum += i;
        }

        squareOfSum = sum * sum;

        System.out.format("%d%n", squareOfSum - sumOfSquares);
    }
}

File: nim/006/euler.nim
var sum_of_squares = 0

var sum = 0

var square_of_sum = 0

for i in 1..100:
  sum_of_squares = sum_of_squares + i * i

  sum = sum + i

square_of_sum = sum * sum

echo(square_of_sum - sum_of_squares)
